eisenhower-matrix/src/main/webapp/index.html

```diff
@@ -26,29 +26,34 @@ <h1>Eisenhower Matrix Scheduler</h1>
       </p>
       <div id="taskForm">
         <div id="createForm">
-          <form action="#">
-            <div class="form-group required">
```

Comment:
Looks good! Did you mean to remove the bootstrap classes for the form? (form-group, form-control)? Just wanted to check :)

Reply:
oops, that was a mistake, thanks for catching it!